app/src/main/java/com/iic/shopingo/ui/request_flow/activities/CreateShoppingListActivity.java

```diff
@@ -10,11 +10,14 @@
 import butterknife.InjectView;
 import butterknife.OnClick;
 import com.iic.shopingo.R;
+import com.iic.shopingo.dal.models.BaseRequest;
+import com.iic.shopingo.dal.models.OutgoingRequest;
+import com.iic.shopingo.dal.models.ShoppingList;
 import com.iic.shopingo.ui.request_flow.views.CreateRequestItemListView;
 import java.util.Currency;
 import java.util.Locale;
 
-public class CreateRequestActivity extends ActionBarActivity {
+public class CreateShoppingListActivity extends ActionBarActivity {
```

Comment:
I thought it made more sense, since this activity contains only the parameters for the shopping list. I can change it back if you see fit.